docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -691,6 +692,22 @@ EXAMPLE:
    This is forced to ``True`` if conda-build is installed and older than 3.18.3,
    because older versions of conda break when conda feeds it the new file format.
 
+.. _set-default-env:
+Specify a default environment to activate on launch
+---------------------------------------------------
+Conda 23.5.0 introduced the ability to set a default environment on shell start-up. This is done
+by setting the ``default_start_environment`` variable.
+
+EXAMPLE:
+
+.. code-block:: yaml
+
+  default_start_environment: my-favourite-env
+
+.. note::
+  If ``default_start_environment`` is not set (or is set and subsequently removed), conda 
```

Comment:
```suggestion
  If ``auto_activate_environment`` is not set (or is set and subsequently removed), conda 
```

Reply:
It wouldn't let me commit on here so have done so locally on my machine. Will push shortly.